packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -185,14 +211,47 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
         />
       </Stack>
     ),
-    [itemHovered, menuTitle, ids.participantItemMenuButton]
+    [itemHovered, strings.menuTitle, ids.participantItemMenuButton]
   );
 
   const onDismissMenu = (): void => {
     setItemHovered(false);
     setMenuHidden(true);
   };
 
+  /* @conditional-compile-remove(one-to-n-calling) */
+  /* @conditional-compile-remove(PSTN-calls) */
+  const participantStateString = React.useMemo((): string | undefined => {
```

Comment:
This is a simple `switch` statement that returns a `string`, so I wouldn't `useMemo` here.

Reply:
Because, this is used inside another useMemo, we need to memoize this function too.

Reply:
Converted it to shorthand if else statement.